packages/core/src/RatingBadge/RatingBadge.js

```diff
@@ -15,7 +15,7 @@ const RatingBadge = styled(Box).attrs(({color, bg}) => ({
 RatingBadge.defaultProps = {
   fontWeight: 'bold',
   px: 2,
-  color: 'white',
+  color: 'alert',
   bg: 'orange',
```

Comment:
should we get rid of `bg` too?

Reply:
I think we'd want to leave it until we actually remove the `bg` prop just to be safe

Reply:
Just confirming to be safe, the text within the RatingBadge will always be white unless styled differently, correct?

Reply:
I don't think it'll alway necessarily be white, e.g. `<RatingBadge color="background.lightest">7.6</RatingBadge>`, but the color prop will behave the same as it does on `Box`.

Reply:
this is actually a breaking change. should we tee it up for v5?